### PR TITLE
Add endpoint for OGN staking stats

### PR DESCRIPTION
**Pull request by shahthepro**

### CODE REVIEW

File: run.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+source ./eagle-python/bin/activate
```
Comment (DanielVF) [inline run.sh:1]: Nice

**Review by DanielVF — APPROVED** [review]
Looks good to me, @shahthepro!

File: eagleproject/core/views.py [hunk 2]
```diff
@@ -285,3 +287,26 @@ def _get_trailing_apy():
...
+def staking_stats(request):
+    with connection.cursor() as cursor:
+        query = """
+        select count(*) as count, sum(staked_amount) as total_staked from (
+            select user_address, sum(
+                case 
+                when is_staked then amount
+                else amount * -1
+                end
+            ) as staked_amount from core_ognstaked group by user_address
```
Comment (mikeshultz) [inline eagleproject/core/views.py:300]: Not sure about this pattern.  Anyone who's withdrawn their stakes will show as negative staked_amounts.  Maybe doesn't need to be dealt with now if it works for you for the dapp.

File: eagleproject/core/models.py [hunk 3]
```diff
@@ -160,3 +160,11 @@ class OusdTransfer(models.Model):
     from_address = models.CharField(max_length=42, db_index=True)
     to_address = models.CharField(max_length=42, db_index=True)
     amount = models.DecimalField(max_digits=64, decimal_places=18, default=0)
+
+class OgnStaked(models.Model):
+    tx_hash = models.CharField(max_length=66)
+    log_index = models.CharField(max_length=66, db_index=True)
+    block_time = models.DateTimeField(db_index=True)
+    user_address = models.CharField(max_length=42, db_index=True)
+    is_staked = models.BooleanField()
```
Comment (mikeshultz) [inline eagleproject/core/models.py:169]: Name implies staked.

**Review by mikeshultz — APPROVED** [review]
Couple comments but if this works for you all I'll err on the side of speed.  Will merge this and try to get this deployed by EOD.

File: eagleproject/core/views.py (hunk 2 above)
Comment (shahthepro) [reply eagleproject/core/views.py:300]: I'm taking a sum and ignoring rows where sum <= 0. So, yeah, it works. Will see if there is a better way to do it.

File: eagleproject/core/models.py (hunk 3 above)
Comment (shahthepro) [reply eagleproject/core/models.py:169]: Model name?

**Pull request merged**